vowpalwabbit/learner.h

```diff
@@ -120,6 +128,7 @@ void generic_driver(const std::vector<vw*>& alls);
 void generic_driver_onethread(vw& all);
 
 inline void noop_sl(void*, io_buf&, bool, bool) {}
+inline void noop_pm(void*, std::vector<std::tuple<std::string, size_t>>&) {}
```

Comment:
I don't know what `noop_sl` is and someone reading this out of context might not know what `noop_pm` is, should we make it `noop_persist_metrics`?